### PR TITLE
Prioritize Cancer Gene Census

### PR DESCRIPTION
**Pull request by aacic**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [ ] Tests: added and/or passed unit and integration tests, or N/A
- [ ] Todos: commented or documented, or N/A
- [ ] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/plot.disco.js [hunk 1]
```diff
@@ -73,7 +73,12 @@ export default async function (termdbConfig, dslabel, sample, holder, genomeObj)
 					{
 						chartType: 'Disco',
 						subfolder: 'disco',
-						extension: 'ts'
+						extension: 'ts',
+						overrides: {
+							label: {
+								showPrioritizeCancerGenes: !!genomeObj.geneset
```
Comment (xzhou82) [inline client/plots/plot.disco.js:79]: works now.
cancer gene census is just one gene set to prioritize the labels, there are many other choices. thus it's not good to call the attribute `CancerGenes` but should be a generic label e.g. `prioritizeGeneLabelsByGeneSets`
same for the folder and script names

Comment (aacic) [reply client/plots/plot.disco.js]: I've changed it and pushed.

**Review by karishma-gangwani — APPROVED** [review]
thanks Alex. would be nice to add comments in the scripts. and do we clean up the legacy disco plot files @xzhou82

**Comment by xzhou82** [comment]
thanks i will review later tonight. no we no longer maintain the disco_old files

**Review by xzhou82 — APPROVED** [review]
thanks

**Pull request merged**